src/Bonsai.ML.HiddenMarkovModels.Design/Bonsai.ML.HiddenMarkovModels.Design.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Title>Bonsai.ML.HiddenMarkovModels.Design</Title>
+    <Description>A package for visualizing data from the Bonsai.ML.HiddenMarkovModels package.</Description>
+    <PackageTags>Bonsai Rx ML Machine Learning Hidden Markov Models Design</PackageTags>
+    <TargetFramework>net472</TargetFramework>
+    <UseWindowsForms>true</UseWindowsForms>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Bonsai.Core" Version="2.8.2" />
+  </ItemGroup>
```

Comment:
```suggestion
```

You should be able to remove this package reference since you will get the right version from the project reference to `Bonsai.ML.Design`. This way you won't have to update the dependency version in two places.